packages/maskbook/src/manifest.json

```diff
@@ -2,7 +2,18 @@
     "name": "Mask Network",
     "version": "1.30.5",
     "manifest_version": 2,
-    "permissions": ["storage", "downloads", "webNavigation", "activeTab"],
+    "permissions": [
+        "storage",
+        "downloads",
+        "webRequest",
+        "webRequestBlocking",
+        "webNavigation",
+        "activeTab",
+        "https://gitcoin.co/*",
+        "https://web-api.coinmarketcap.com/*",
+        "https://v.cent.co/*",
+        "https://api-v4.zerion.io/*"
```

Comment:
Please don't. This is blocking the extension to be auto-updated. Those permissions should be requested dynamically.